packages/embark/src/cmd/cmd_controller.js

```diff
@@ -388,6 +411,14 @@ class EmbarkController {
       function (callback) {
         engine.init({}, callback);
       },
+      function (callback) {
+        engine.startService("libraryManager").installAll((err) => {
+          if (err) {
+            return callback(err);
+          }
+          callback();
+        });
+      },
```

Comment:
I assume we're still experiencing race conditions if we don't explicitly run this before other services are started?

Reply:
Yes, definitely. For example, `web3` with a version that doesn't match embark's will start installing but other services will move on and there will be stack traces in embark's output.